interface/python_binding/tests/test_ktx_texture.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023, Shukant Pal and Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from pyktx import *
+from test_config import __test_images__
+import unittest
+
+
+class TestKtxTexture(unittest.TestCase):
```

Comment:
How are the tests run? Are they always run as part of the build? I didn't notice anything in the CI additions.

Reply:
They are automatically run as a part of the build. See `python_binding/build.py`.  These tests are added to CTest in the python_binding/CMakeLists.txt file.